### PR TITLE
Tech/cc 26886 add Launch Darkly sync actions

### PR DESCRIPTION
**Pull request by ralphroper**

### CODE REVIEW

File: launchdarkly-segments-sync/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "keywords": [
+    "GitHub",
+    "Actions",
+    "JavaScript"
+  ],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@actions/core": "^1.10.0",
+    "@actions/github": "^5.1.1",
```
Comment (aarontwf) [inline launchdarkly-segments-sync/package.json]: Doesn't look like this one is used so can remove

File: launchdarkly-flags-sync/package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  "keywords": [
+    "GitHub",
+    "Actions",
+    "JavaScript"
+  ],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@actions/core": "^1.10.0",
+    "@actions/github": "^5.1.1",
```
Comment (aarontwf) [inline launchdarkly-flags-sync/package.json]: Doesn't look like this one is used so can remove

**Review by aarontwf — APPROVED** [review]

File: launchdarkly-flags-sync/package.json (hunk 2 above)
Comment (ralphroper) [reply launchdarkly-flags-sync/package.json]: removed

File: launchdarkly-flags-sync/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+### flag (optional)
+Sync only the specified flag. By default all flags are synced.
+
+### tag (optional)
+Sync flags with the given tag(s). Only flags with all tags will sync
+
+### omitSegments (optional)
+Omit segments when syncing. A sync may fail if it dependenet on a segment existing in the 
+destination environment. If this is the case then run launchdarkly-segments0sync action beforehand.
```
Comment (blake-cartoncloud) [inline launchdarkly-flags-sync/README.md]:
```suggestion
destination environment. If this is the case then run launchdarkly-segments-sync action beforehand.
```

**Review by blake-cartoncloud — APPROVED** [review]

File: launchdarkly-segments-sync/package.json (hunk 1 above)
Comment (ralphroper) [reply launchdarkly-segments-sync/package.json]: removed

**Pull request merged**